bindings/rust/wasmedge-sys/src/config.rs

```diff
@@ -310,6 +324,29 @@ impl Config {
     pub fn is_time_measuring(&self) -> bool {
         unsafe { wasmedge::WasmEdge_ConfigureStatisticsIsTimeMeasuring(self.ctx) }
     }
+
+    /// Enables or Disables the `Interruptible` option of AOT compiler.
+    ///
+    /// This option determines to generate interruptible binary or not when compilation in AOT compiler.
+    ///
+    /// # Argument
+    ///
+    /// - `flag` specifies if turn on the `Interruptible` option.
+    pub fn interruptible(self, enable: bool) -> Self {
+        unsafe {
+            if (enable && !self.interruptible_enabled())
+                || (!enable && self.interruptible_enabled())
+            {
+                wasmedge::WasmEdge_ConfigureCompilerSetInterruptible(self.ctx, enable);
```

Comment:
`Interruptible` is the same as `wasi`. Just set the value. No need to check. Thanks.

Reply:
Thanks for your reminder, I forget to fix it, sorry about that.